packages/core/src/components/carousel/stories/style.css

```diff
@@ -24,3 +24,12 @@
   justify-content: center;
   width: 100% !important;
 }
+
+.item-smaller {
+  align-items: center;
+  background: coral;
+  display: flex;
+  height: 400px;
+  justify-content: center;
+  width: 250px;
+}
```

Comment:
(question) do you want to simulate small screens in carousel? in theory we could just use the responsive tool in storybook

<img width="774" alt="image" src="https://github.com/juntossomosmais/atomium/assets/3603793/83b599b3-bc6d-4733-8fd0-23539613663d">

Reply:
No. The idea is showing when the content is smaller than slider-item. I created Center Item Content 
![Screen Shot 2024-02-22 at 15 24 56](https://github.com/juntossomosmais/atomium/assets/37077860/4b8c9c5c-4521-4627-b563-6d6be3349f02)